### DISCUSSION TITLE
ELNc Meeting 3

### DISCUSSION THREAD

**Discussion by NicolasCARPi** [TheELNConsortium/TheELNFileFormat#39] [General]
Hello everyone,

We want to make another meeting sooner rather than later, because it gets the ball rolling!

Here is an embryo of agenda:

* validator : what do we want to enforce in the .eln: need to have a prototype to ask what makes sense. Multi level: 1 ro crate. 2 check terms and stuff.
* author present or not (using a _ in author field vs removing the field)
* which terms can we agree upon?
* what problems do we encounter when importing different .eln from different ELN producing software
* can we have a Gold standard experiment with a given set of parameters/data that we then try all to import/export?
* Dataverse integration (along with ro-crate)
* new members suggestions?

Please take the time to indicate your disponibility on this poll:

https://framadate.org/GneiClvEBUQgOhYM

I've chosen 16:00 CET so people in the US can join, too.

@amirgolp
@bronger
@CEschke
@djeanner
@FlorianRhiem
@fspreck-indiscale
@jmanideep
@kirchner
@markus1978
@martin-held-hereon
@mikee1976
@mKowalski256
@nicobrandt
@NicolasCARPi
@nicolejung
@pdurbin
@quazgar
@salexan2001
@sayedahmad
@SteffenBrinckmann
@TheRich-71
@twindiscale

(`jq '.[] | "@" + .login' eln-org.json | tr -d '"'`)

Feel free to propose discussion points here, too!

**Comment by NicolasCARPi** [comment]
Current date would be the 12th of December, 4pm CET.

ping @amirgolp
@bronger
@CEschke
@djeanner
@FlorianRhiem
@fspreck-indiscale
@jmanideep
@kirchner
@markus1978
@martin-held-hereon
@mikee1976
@mKowalski256
@nicobrandt
@NicolasCARPi
@nicolejung
@pdurbin
@quazgar
@salexan2001
@sayedahmad
@SteffenBrinckmann
@TheRich-71
@twindiscale

**Comment by pdurbin** [comment]
I'm not sure if you want us to comment with regrets, but I can't make it since it conflicts with the [Dataverse community call](https://dataverse.org/community-calls). I appreciate the US-friendly time though!

By the way, I mentioned the .eln format today during a call with people from https://github.com/mlcommons/croissant 😄

**Reply by NicolasCARPi** [reply]
:croissant:

**Comment by NicolasCARPi** [comment]
Hello everyone,

Chosen date is December 12th at 4PM CET. Mark it on your calendar. Link will be shared here. Looking forward to seeing you there!

ping @amirgolp
@bronger
@CEschke
@djeanner
@FlorianRhiem
@fspreck-indiscale
@jmanideep
@kirchner
@markus1978
@martin-held-hereon
@mikee1976
@mKowalski256
@nicobrandt
@NicolasCARPi
@nicolejung
@pdurbin
@quazgar
@salexan2001
@sayedahmad
@SteffenBrinckmann
@TheRich-71
@twindiscale

**Reply by NicolasCARPi** [reply]
Meeting link: https://pwpush.com/en/p/frshv4u2kr4riw

**Comment by NicolasCARPi** [comment]
ELNc Meeting 3 :: It's in 20 minutes:

Zoom link:  https://pwpush.com/en/p/frshv4u2kr4riw

ping @amirgolp
@bronger
@CEschke
@djeanner
@FlorianRhiem
@fspreck-indiscale
@jmanideep
@kirchner
@markus1978
@martin-held-hereon
@mikee1976
@mKowalski256
@nicobrandt
@NicolasCARPi
@nicolejung
@pdurbin
@quazgar
@salexan2001
@sayedahmad
@SteffenBrinckmann
@TheRich-71
@twindiscale

**Comment by NicolasCARPi** [comment]
Here are the minutes:

General information
===================

Meeting was held on Tuesday December 12th at 4 PM CET on Zoom.

Participants present to the meeting
===================================

*   [**OpenBIS**](https://openbis.ch/) (ETH Zürich): Adam Laskowski and Juan Fuentes
*   [**Indiscale**](https://www.indiscale.com/) (German RDM company): Thomas Weiβ and Florian Spreckelsen
*   [**FAIRMAT Nomad**](https://nomad-lab.eu/nomad-lab/) (German open source ELN for material sciences): Amir Golparvar
*   [**SampleDB**](https://github.com/sciapp/sampledb) (German open source ELN): Florian Rhiem
*   [**iRODS Consortium**](https://irods.org/about/): Kory Draugshn and Terell Russell (involved with RSpace and Dataverse integrations)
*   [**Kadi4Mat**](https://kadi.iam.kit.edu/) (German open source ELN): Manideep and Nico Brandt
*   [**Chemotion**](https://chemotion.net/) (German open source ELN): Martin Starman
*   [**PASTA**](https://github.com/PASTA-ELN/pasta-eln) (German open source ELN): Steffen Brinckmann
*   [**eLabFTW**](https://www.elabftw.net) (French open source ELN): Nicolas Carpi

Minutes of the meeting
======================

Discussion on terms
-------------------

We want to agree on the attributes that we will most commonly find describing a node in a json-ld of the metadata of the ro-crate in the .eln archive.

*   Common terms that we can agree upon, to guide also new users
    *   name
    *   author (unless GDPR clash)
    *   created at
    *   modified at
    *   comments
    *   mentions
    *   identifier: just a string, different for each ELN: ORCID for an Author
    *   text is for the content
    *   description is for describing what is this thing and should not be used for user data
    *   keywords (tags): must be a string! we previously used an array, but it must be a comma separated list!
*   author: PII: empty for GDPR compliance? Don't add the Author property at all.

Question about mandatory sha256sum
----------------------------------

Is anyone against having mandatory sha256sum?

Nico Brandt: might not be a good idea, especilally for bigger files, we might not have it.

Spec (modification to be done):

sha256 not mandatory, should be there

if there: must be checked: if not match: error

if no there, nothing to do

RO-Crate 1.2 coming up
----------------------

[https://www.researchobject.org/ro-crate/1.2-DRAFT/appendix/changelog.html](https://www.researchobject.org/ro-crate/1.2-DRAFT/appendix/changelog.html)

Need to look if anything would break, at first glance it looks ok.

Validator
---------

Validator: existing github action that needs to be improved with what we discussed.

Should be usable from CLI to ease development cycle. It is already because it's a few lines of python (but not really standalone either).

Gold standard
-------------

The idea is to have an experiment that is the same in all examples in the repo, so we can easily compare to each other because the content is exactly the same.

Propositions for new members in the Consortium
----------------------------------------------

Zenodo would be a good candidate.

Updates on individual ELNs
--------------------------

Openbis: several projects with interoperability, planning to get something next year. Suggestion: use from CLI for validator.

Ideas for the future
--------------------

Florian: we have no way of knowing that the .eln has not been tempered with. Could be nice to have a way to sign it.

Nico C: it could be a .well-known/ file on the source webserver that serves the public key of the instance

Should be on the agenda of next meeting.

Meetings recurrence
-------------------

Time between elnc meetings: 3 months

**Next meeting March 11th 2024 at 4 PM CET**

**Reply by trel** [reply]
spelling corrections - Kory Draughn and Terrell Russell
